packages/feedback/src/util/sendFeedbackRequest.ts

```diff
@@ -34,58 +34,28 @@ export async function sendFeedbackRequest({
     type: 'feedback',
   };
 
+  // Create a new scope
+  const scope = hub.pushScope();
```

Comment:
I'd recommend to use `withScope` instead (we'll probably get rid of `pushScope` in v8):

```js
withScope((scope) => {
  scope.clearBreadcrumbs(); 

    if ([FEEDBACK_API_SOURCE, FEEDBACK_WIDGET_SOURCE].includes(String(source))) {
      scope.setLevel('info');
    }

    const feedbackEvent = ...
});
```

Reply:
Updated using `withScope`, had to create a promise because I need the return value inside of `withScope` callback